src/Kliva/ViewModels/SidePaneViewModel.cs

```diff
@@ -78,8 +79,6 @@ public MenuItem SelectedTopMenuItem
                                 ClubsCommand.Execute(null);
                                 break;
                         }
-
-                        SelectedTopMenuItem = null;
```

Comment:
Not sure what these were doing for us. (Didn't seem to be reflected in the selection visual.)